sql/create_train_val.go

```diff
@@ -85,7 +85,7 @@ func createMaxcomputeRandomTable(target, slct string, db *DB) error {
 		return e
 	}
 	// create a table, then split it into train and val tables
-	stmt := fmt.Sprintf("CREATE TABLE %s LIFECYCLE %d AS SELECT *, RAND() AS %s FROM (%s) AS %s_ori", target, temporaryTableLifecycle, randomColumn, slct, target)
+	stmt := fmt.Sprintf("CREATE TABLE %s LIFECYCLE %d AS SELECT *, RAND(42) AS %s FROM (%s) AS %s_ori", target, temporaryTableLifecycle, randomColumn, slct, target)
```

Comment:
@sperlingxx Please: 

1. define the ultimate number 42 as a const value.
2. add comments why need to fix the random seed for better readability.